Utils.py

```diff
@@ -84,3 +89,12 @@ def get(self):
         self.check.remove(item)
         return item
 
+# ============== Logging Setup
+
+
+def SetLoggerUname(uname):
```

Comment:
rename to `setLoggerUname`, by convention functions do not start with a capital letter